public/app/core/time_series2.ts

```diff
@@ -107,6 +108,7 @@ export default class TimeSeries {
     this.valueFormater = kbn.valueFormats.none;
     this.stats = {};
     this.legend = true;
+    this.hideTooltip = false;
```

Comment:
not sure boolean instance variables needs to be initialized to false

Reply:
without initialization, the value is `undefined`.
I like boolean initialized value, but other boolean variables are not initialized, so I remove the initialization.